### PR TITLE
added the below 5 javascript files:

### PR DESCRIPTION
**Pull request by DeepthiDukka**
mainweek3.js
travelTime.js
seriesDuration.js
songsPlaylist.js
notes.js

### CODE REVIEW

File: javascript/javascript1/week3/mainweek3.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*Item array removal */
+
+const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
+const nameToRemove = 'Ahmad';
+
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    console.log("removed item: " + names[i]);
+    names.splice(1, i);
```
Comment (aleksd-dfds) [inline javascript/javascript1/week3/mainweek3.js]: actually i think `names.splice(i,1)` is correct. Try it with nameToRemove = 'kristina'

File: javascript/javascript1/week3/mainweek3.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  }
+}
+console.log(names);
+
+/* Another way of removing 'Ahmad' from the array */
+names.indexOf("Ahmad");
```
Comment (aleksd-dfds) [inline javascript/javascript1/week3/mainweek3.js]: ok but you're not using the index you found?

File: javascript/javascript1/week3/mainweek3.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+names.splice(1, 1);
```
Comment (aleksd-dfds) [inline javascript/javascript1/week3/mainweek3.js]: how do you know the index is 1?

File: javascript/javascript1/week3/seriesDuration.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    let totalDays = totalHoursToDays + seriesDurations[i].days;
+    let totalYears = Number((totalDays / 365).toFixed(3));
+    let avgLifeSpan = 80;
+    let percentForEachSeries = Number((totalYears * 100 / avgLifeSpan).toFixed(2));
+    seriesDurations[i].years = percentForEachSeries;
+    let titlesOfEachSeries = seriesDurations[i].title;
+    console.log((`${titlesOfEachSeries} took ${percentForEachSeries}% of my life`));
+    total += seriesDurations[i].years;
+
+}
```
Comment (aleksd-dfds) [inline javascript/javascript1/week3/seriesDuration.js:37]: looks good. Maybe extract the for loop into a function also.

File: javascript/javascript1/week3/travelTime.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function timeTakenForDestination(travelInformation) {
+
+  let time = travelInformation.destinationDistance / travelInformation.speed;
+  let totalHours = Math.floor(time);
+  let totalMinutes = Math.round((time - totalHours) * 60);
+  return `${totalHours} hours and ${totalMinutes} minutes`
```
Comment (aleksd-dfds) [inline javascript/javascript1/week3/travelTime.js:12]: it's better to return an object with {totalHours, totalMinutes} and not a message. You can construct that message inside console.log from the object.

**Review by aleksd-dfds — COMMENTED** [review]
Nicely done, try to implement the feedback and i'll look over it again.

File: javascript/javascript1/week3/mainweek3.js (hunk 1 above)
Comment (DeepthiDukka) [reply javascript/javascript1/week3/mainweek3.js]: Feedback implemented here

File: javascript/javascript1/week3/seriesDuration.js (hunk 4 above)
Comment (DeepthiDukka) [reply javascript/javascript1/week3/seriesDuration.js:37]: Extracted the code into function now.

**Review by aleksd-dfds — APPROVED** [review]

**Pull request merged**